src/schema/rules/sidecars/nirs.yaml

```
@@ -0,0 +1,138 @@
+---
+NIRSCoordsystemBase:
+  selectors:
+    - datatype == "nirs"
+    - suffix == "coordsystem"
+  fields:
+    NIRSCoordinateSystem: required
+    NIRSCoordinateUnits: required
+    NIRSCoordinateProcessingDescription: recommended
+    FiducialsDescription: optional
+    FiducialsCoordinates: recommended
+    FiducialsCoordinateSystem: recommended
+    FiducialsCoordinateUnits: recommended
+    AnatomicalLandmarkCoordinates: recommended
+    AnatomicalLandmarkCoordinateSystem: recommended
+    AnatomicalLandmarkCoordinateUnits: recommended
+
+NIRSCoordinateSystemDescriptionRec:
+  selectors:
+    - datatype == "nirs"
+    - suffix == "coordsystem"
+    - json.NIRSCoordinateSystem != "other"
+  fields:
+    NIRSCoordinateSystemDescription:
+      level: recommended
+      level_addendum: required if NIRSCoordinateSystem is "other"
+
+NIRSCoordinateSystemDescriptionReq:
+  selectors:
+    - datatype == "nirs"
+    - suffix == "coordsystem"
+    - json.NIRSCoordinateSystem == "other"
+  fields:
+    NIRSCoordinateSystemDescription: REQUIRED
+
+AnatomicalLandmarkCoordinateSystemDescriptionRec:
+  selectors:
+    - datatype == "nirs"
+    - suffix == "coordsystem"
+    - json.AnatomicalLandmarkCoordinateSystem != "other"
+  fields:
+    AnatomicalLandmarkCoordinateSystemDescription:
+      level: recommended
+      level_addendum: required if NIRSCoordinateSystem is "other"
+
+AnatomicalLandmarkCoordinateSystemDescriptionReq:
+  selectors:
+    - datatype == "nirs"
+    - suffix == "coordsystem"
+    - json.AnatomicalLandmarkCoordinateSystem == "other"
+  fields:
+    AnatomicalLandmarkCoordinateSystemDescription: required
+
+FiducialsCoordinateSystemDescriptionRec:
+  selectors:
+    - datatype == "nirs"
+    - suffix == "coordsystem"
+    - json.FiducialsCoordinateSystem != "other"
+  fields:
+    FiducialsCoordinateSystemDescription:
+      level: recommended
+      level_addendum: required if FiducialsCoordinateSystem is "other"
+
+FiducialsCoordinateSystemDescriptionReq:
+  selectors:
+    - datatype == "nirs"
+    - suffix == "coordsystem"
+    - json.FiducialsCoordinateSystem == "other"
+  fields:
+    FiducialsCoordinateSystemDescription: required
+
+FnirsBase:
+  selectors:
+    - datatype == "nirs"
+    - suffix == "nirs"
+  fields:
+    TaskName: required
+    InstitutionName: recommended
+    InstitutionAddress: recommended
+    Manufacturer: recommended
+    ManufacturersModelName: recommended
+    SoftwareVersions: recommended
+    TaskDescription: recommended
+    Instructions: recommended
+    CogAtlasID: recommended
+    CogPOID: recommended
+    DeviceSerialNumber: recommended
+    RecordingDuration: recommended
+    HeadCircumference: recommended
+    HardwareFilters: recommended
+    SubjectArtefactDescription: recommended
+
+FnirsRequired:
+  selectors:
+    - datatype == "nirs"
+    - suffix == "nirs"
+  fields:
+    CapManufacturer:
+      level: recommended
+      level_addendum: |
```

These all seem more like description addenda than requirement level addenda.

```suggestion
      addendum: |
```

---

I'm currently seeing only `level_addendum` and `description_addendum` being used in sidecars rules, is there a move towards just `addendum`?

---

Yeah these three are better as description addendums